### PR TITLE
Mobile branch

### PR DESCRIPTION
**Pull request by NelsonAraujo93**
In this branch i realized the next activities:

1. Add index.html
2. Add style.css
3. Add index.js
4. Add and import assets folder
5. Add Toolbar and Headline section in Index.html
6. Add Rules in style.css (Flexbox)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <header>
+        <section class="mobile-nav">
+            <div class="landing-header">
+                <div class="header-image"></div>
+                <button id='open-menu-btn' class="menu-btn">
+                    <img src="./assets/ic_menu.svg">
+                </button>
```
Comment (DuaneDave) [inline index.html]: For semantic reasons and to deliver a great user experience on mobile devices, especially for screen readers. I suggest you make the wrapper for this hamburger menu a `nav` tag

Comment (DuaneDave) [inline index.html]: [OPTIONAL] In line with the need for quality UX, I also suggest you add an aria-control attribute to the wrapper to make it clear to screen readers that contents are hidden. see [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls#:~:text=The%20aria%2Dcontrols%20attribute%20identifies%20the%20element%20(or%20elements)%20whose%20contents%20or%20presence%20are%20controlled%20by%20the%20element%20on%20which%20the%20attribute%20is%20set%2C%20regardless%20of%20what%20type%20of%20interaction%20initiates%20the%20impacted%20behavior.)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                        <a>Contact</a>
+                    </li>
+                </ul>
+                <div class="bottom-image"></div>
+            </nav>
+        </section>
+    </header>
+    <main>
+        <section class="introduction-mobile">
+            <h1>Nelson Araujo</h1>
```
Comment (DuaneDave) [inline index.html:44]: According to the provided template, you were supposed to be using `Playfair display` as a choice `font-family` but you are using `Poppins`.
I recommend using the choices `font-family` to keep your UI consistent with the Figma design so your website UI will be impeccable

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            <p>Hello! and welcome to my website. I am a web developer and why not your future collaborator? Take some minutes to look at my works. And if you want to know more about me, do not hesitate contacting.</p>
+            <button class="main-btn">
+                Start collaborating
+            </button>
+        </section>
+        <div class="deco-image"></div>
```
Comment (DuaneDave) [inline index.html]: To stay in sync with the template, I recommend making this image a full-width one according to the design

Your design
std pic

Figma design
fig width

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+.menu-btn {
+  width: 20px;
+  height: 20px;
+  background-color: transparent;
+  border: none;
+}
+
+#menu-container {
+  height: 100vh;
+  width: 100%;
```
Comment (DuaneDave) [inline styles.css]: [OPTIONAl] I can see that `width: 100%` is a very common rule you declared all through your CSS. In order to not repeat yourself, consider making a separate class for this rule

Like this.

```
.full-width { 
    width: 100%;
}
```

You can then apply this in your HTML tags

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @NelsonAraujo93  👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Solid design 💯 
✔️ Professional commit messages 👏 
✔️ All linters passing 💯 
✔️ Intuitive use of HTML tags 💯 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
I left some optional suggestions in the comment below

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (NelsonAraujo93) [reply index.html]: Hello @DuaneDave, thanks for the advices, the image is actually set as full-width, but will change the background-size property from 'contain' to 'cover'.

**Comment by NelsonAraujo93** [comment]
Hey @DuaneDave ,
I just finished and did a commit with the changes you asked me for. Thank you so much, because i learned about UX and better practices with your comments, let me know is there are more things to enhance.

**Comment by DuaneDave** [comment]
If all are applied, that will be it.

Good job man 👍

**Comment by NelsonAraujo93** [comment]
Hello @DuaneDave ,
What is the next step? should I merge the project or request a new pull? I am asking this because in my dashboard the activity state is stilling in 'change request'.

**Review by jerryowusu — APPROVED** [review]
Hi @NelsonAraujo93 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
Excellent communication with reviewer :wink: 
Good commit messages :heavy_check_mark: 
All linters are passing :100: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — COMMENTED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @NelsonAraujo93 , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**